modules/mod_ginger_edit/templates/logon_error/message.pw.tpl

```diff
@@ -0,0 +1,3 @@
+<p>
+    {_ You entered an unknown username or password.  Please try again. _}
```

Comment:
Should be a single space.

Reply:
Removed (came from previous ginger text)